conda/plugins/subcommands/doctor/health_checks.py

```diff
@@ -163,8 +160,16 @@ def env_txt_check(prefix: str, verbose: bool) -> None:
     print(f"Environment listed in environments.txt file: {present}\n")
 
 
+def requests_ca_bundle_check(prefix: str, verbose: bool) -> None:
+    if (path := os.getenv("REQUESTS_CA_BUNDLE")) and not Path(path).exists():
```

Comment:
Yeah. I was in two minds about this when I wrote the `REQUESTS_CA_BUNDLE` check. 
Displaying a "health check passes ✅ " message is consistent with how we display outputs for other health checks.  
At the same time I thought "REQUESTS_CA_BUNDLE check passes" would confuse users who are not advanced and don't know what `REQUESTS_CA_BUNDLE` is. 🤔 

BUT 
Now that I'm thinking about it again: maintaining consistency is probably a better idea. 👍🏼 
I would change the "check passes" message to "env var `REQUESTS_CA_BUNDLE` points to a valid file. ✅ " so that it's more descriptive for non-advanced users.